impersonate/tests.py

```diff
@@ -45,21 +45,31 @@
 else:
     User = get_user_model()
 
+try:
+    # Django <=1.9
+    from django.conf.urls import patterns
+except ImportError:
+    patterns = None
+
+
+def test_view(request):
+    return HttpResponse('OK {0}'.format(request.user))
 
-urlpatterns = patterns(
-    '',
+
+urls = [
```

Comment:
nit 1: can this be a tuple instead of a list?
nit 2: assign to urlpatterns instead, and remove the else condition of your check below for maximum pythonicness